npm/translatePolicy.go

```diff
@@ -38,6 +38,19 @@ func craftPartialIptEntrySpecFromPort(portRule networkingv1.NetworkPolicyPort, s
 	return partialSpec
 }
 
+func getPortType(portRule networkingv1.NetworkPolicyPort) string {
+	if portRule.Port == nil {
```

Comment:
nit: We can combine 1st and last check 
if portRule.Port == nil || portRule.Port.IntValue() != 0 
{ 
    return "validport" 
} else ...